tests/examples/pacer/nef/arwd.json

```diff
@@ -22,6 +22,9 @@
         "sean@sanfordlawfirm.com"
       ],
       "name": "Josh Sanford"
+    },
+    {
```

Comment:
Should this get pulled into a different key? Amir Zakirali got the notification "by other means," presumably by snail mail, so they're not an `email_recipient`.

Reply:
I guess we should keep the newlines in here too, so we can display these addresses?

Reply:
I am wondering if this is getting a bit out of scope. This is one of the less reliable parts of the parser. Maybe I shove anything that's not an obvious email into "other_recipients" and we can iterate on it in the future.